reporting/test/alive-message-generator.spec.js

```diff
@@ -56,6 +58,9 @@ describe('#AliveMessageGenerator', function () {
       os: 'Linux',
       language: 'en-US',
```

Comment:
is this still needed?

Reply:
The interface provider is not explicitly defined. It is true that the `language` field will be ignored by the implementation. 

But it is also a valid result, and it matches what the provider will return in the production environment. So overall, keeping the same data as it would be in production looks okay to me.